processor/src/errors.rs

```diff
@@ -16,7 +16,9 @@ pub enum ExecutionError {
     DivideByZero(u32),
     DuplicateAdviceKey(Word),
     FailedAssertion(u32),
+    InaccessibleMemoryAddress(u64),
     InvalidFmpValue(Felt, Felt),
+    NonPowerOf2PolyDomainSize(u64),
```

Comment:
nit: would probably rename to something like `NttDomainSizeNotPowerof2`.

Reply:
Addressed in f4e857e1